--- origsrc/shine-2.0.0/Makefile.am	2013-06-11 23:16:32.000000000 -0500
+++ src/shine-2.0.0/Makefile.am	2013-10-29 20:00:46.771690700 -0500
@@ -13,14 +13,14 @@ libshine_la_SOURCES  = src/lib/bitstream
                        src/lib/l3subband.c src/lib/layer3.c src/lib/reservoir.c \
                        src/lib/tables.c
 
-libshine_la_LDFLAGS  = -lm -no-undefined -version-info 2:0:0 -export-symbols libshine.sym
+libshine_la_LDFLAGS  = -lm -no-undefined -version-info 2:0:0 -export-symbols $(srcdir)/libshine.sym
 libshine_ladir       = ${prefix}/include/shine
 libshine_la_HEADERS  = src/lib/layer3.h
 
 bin_PROGRAMS     = shineenc
 shineenc_SOURCES = src/bin/main.c src/bin/wave.c
 shineenc_LDADD   = libshine.la
-shineenc_CFLAGS  = -Isrc/lib
+shineenc_CFLAGS  = -I$(top_srcdir)/src/lib
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = shine.pc
